modules/applying-custom-seccomp-profile.adoc

```diff
@@ -0,0 +1,22 @@
+[id="applying-custom-seccomp-profile_{context}"]
+= Applying the custom seccomp profile to the workload
+
+.Prerequisite
+* The cluster administrator has set up the custom seccomp profile. For more details, see "Setting up the custom seccomp profile".
+
+.Procedure
+* Apply the seccomp profile to the workload by setting the `securityContext.seccompProfile.type` field as following:
++
+.Example
++
+[source, yaml]
+----
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: <custom-name>.json <1>
+----
+<1> Provide the name of your custom seccomp profile.
```

Comment:
`seccompProfile` here as well?

Reply:
Here too it refers to the profile and not the field in the code. Thanks for the catch though.